.github/workflows/ci.yml

```diff
@@ -6,6 +6,8 @@ on:
       - master
     tags:
       - '*'
+  schedule:
+    - cron: '0 0 * * 1' # runs 00:00 UTC on every Monday
```

Comment:
Does a failure here notify anyone within the org @mortenpi?

Reply:
Good point. I think it will notify whoever is the author of the last commit, which might not be ideal.

Reply:
Actually, it looks like it's the person who changes the cron line:

> Notifications for scheduled workflows are sent to the user who last modified the cron syntax in the workflow file. For more information, see "[Notifications for workflow runs](https://docs.github.com/en/actions/monitoring-and-troubleshooting-workflows/notifications-for-workflow-runs)."

Which would be me right now, which seems fine.

- https://docs.github.com/en/actions/writing-workflows/choosing-when-your-workflow-runs/events-that-trigger-workflows#schedule
- https://docs.github.com/en/actions/monitoring-and-troubleshooting-workflows/monitoring-workflows/notifications-for-workflow-runs

Reply:
https://docs.github.com/en/account-and-profile/managing-subscriptions-and-notifications-on-github/setting-up-notifications/configuring-notifications#github-actions-notification-options I think it will notify the people that actively subscribe to e.g. failed runs? I don't think that is turned on by default.

Reply:
Ok, I'm subscribed to failures so between us it should be sufficient.